src/ocean/io/select/client/SelectEvent.d

```diff
@@ -87,11 +87,25 @@ public abstract class ISelectEvent : IAdvancedSelectClient
         Constructor. Creates a custom event and hooks it up to the provided
         event handler.
 
+        Supported OR-combination of `flags`, starting with Linux 2.6.27:
+          - `EFD.NONBLOCK`: Set the `O_NONBLOCK` file status flag on the new
+                open file description.
+          - `EFD.CLOEXEC`: Set the close-on-exec (`FD_CLOEXEC`) flag on the new
+                file descriptor. (See the description of the `O_CLOEXEC` flag
+                in `open(2)` for reasons why this may be useful.)
+          - `EFD.SEMAPHORE` (since Linux 2.6.30): Provide semaphore-like
+                semantics for reads from the new file descriptor.
+
+        Params:
+            initval = initial counter value
```

Comment:
I'd be inclined not to add the initial value setter. No one has ever needed this feature, so I wouldn't complicate the API for no reason.